tests/Functional/ContentRenderTestBase.php

```diff
@@ -205,4 +206,118 @@ protected function createContactEntity(string $name, string $bundle, int $status
     return $contact;
   }
 
+  /**
+   * Asserts Contact entity field rendering.
+   *
+   * @param \Behat\Mink\Element\NodeElement $element
+   *   Rendered element.
+   * @param string $name
+   *   Name of the Contact entity.
+   */
+  protected function assertContactEntityDefaultDisplay(NodeElement $element, string $name):void {
+    $contact_name = $element->findAll('css', 'h3');
+    $this->assertCount(1, $contact_name);
+    $this->assertEquals($name, $contact_name[0]->getText());
+
+    $contact_body = $element->findAll('css', '.ecl-editor');
+    $this->assertCount(1, $contact_body);
+    $this->assertEquals("Body text $name", $contact_body[0]->getText());
+
+    $contacts_html = $element->getHtml();
+    $field_list_assert = new FieldListAssert();
+    $contact_expected_values = [
+      'items' => [
+        [
+          'label' => 'Organisation',
+          'body' => "Organisation $name",
+        ], [
+          'label' => 'Website',
+          'body' => "http://www.example.com/website_$name",
```

Comment:
I can add node as well and this is what I get:
![image](https://user-images.githubusercontent.com/22004498/98805079-5d1d9680-2417-11eb-844a-bcbe5f297729.png)